assets/css/style.css

```diff
@@ -264,6 +264,72 @@ header {
     margin-top: 100px;
     text-align: justify;
 }
+.footer {
+    background-color: #FFE66D;
+    background-size: cover;
+    padding :30px 40px;
+}
+.footer-logo {
+    color: #0e2d31;
+    font-size: 60px;
+    float: right;
+    padding-right: 100px;
+}
+
+.footer-list {
+    color: #1a535c;
+    font-size: 25px;
+}
+.footer-list li{
+    list-style: none;
+    padding-right: 60px;
+    padding-left: 20px;
+    cursor: pointer;
+}
+.Beranda {
+    float: left;
+
+}
+.Gejala {
+    float: left;
+}
+.Pencegahan {
+    float: left;
+}
+.call-center{
```

Comment:
CSS class is used to avoid repetitive use of property. For example, `float: left;`

You can remove class `beranda`, `gejala`, and `pencegahan` and use one class instead. Make sure the change is reflected to your index.html too.

assets/css/style.css
```
.left {
    float: left;
}
```

index.html
```
        <footer class="footer">
            <div class="footer-logo">InCov</div>
            <div class="footer-list"> 
            <div class="footer-container">
                <ul>
                    <li href="#" class="left"> Beranda </li>
                    <li href="#" class="left"> Gejala </li>
                    <li href="#" class="left"> Pencegahan </li>
                    <li href="#" class="left"> Jenis Pasien </li>
                </ul>
...
```